### PR TITLE
Feature/2325 few touchups for experience accordions

### LINKED ISSUE #2325: Few touchups For Experience Accordions
h/t @brindasasi noticed a few touchups needed for experience accordions:
1. Date format should be MMM YYYY
2. Skill titles should be in the primary colour

### PR DESCRIPTION
**Pull request by RVany**
Resolves #2325 
h/t @brindasasi noticed a few touchups needed for experience accordions:
1. Date format should be MMM YYYY
2. Skill titles should be in the primary colour

### CODE REVIEW

File: frontend/common/src/fakeData/fakeExperiences.ts [hunk 1]
```diff
@@ -58,6 +58,11 @@ const sampleSkill2: Skill = {
   name: theSkillString2,
   experienceSkillRecord: theExperienceSkillRecord,
 };
+function formattedDeate(date: string) {
```
Comment (vd1992) [inline frontend/common/src/fakeData/fakeExperiences.ts:61]: Just going to be a bit picky, I assume you meant _formattedDate_

**Comment by vd1992** [comment]
I think you forgot to include the
"Skill titles should be in the primary colour" 
bit in the issue. Skills do not appear highlighted

File: frontend/common/src/fakeData/fakeExperiences.ts (hunk 1 above)
Comment (RVany) [reply frontend/common/src/fakeData/fakeExperiences.ts:61]: you are right!

**Comment by RVany** [comment]
> I think you forgot to include the "Skill titles should be in the primary colour" bit in the issue. Skills do not appear in the purple text

I think I added the black color. 
image
@vd1992 @petertgiles It should be purple?

**Comment by RVany** [comment]
@petertgiles Can you please share the wireframe? It looks like the bullet is misplaced and the Date Format is 
image
or 
image

**Comment by vd1992** [comment]
Nice! Got those other things sorted out. 
Unfortunately, SkillAccordion stories appear to have broken as a result of changes to fakeExperiences 

image

**Comment by RVany** [comment]
> Nice! Got those other things sorted out. Unfortunately, SkillAccordion stories appear to have broken as a result of changes to fakeExperiences
> 
> ![image](https://user-images.githubusercontent.com/40485260/163067157-cbc244df-3c6c-4541-bc66-f8d7a97bf623.

looks like woring in local. I will check again and didn't understand the reason you mentioned. Good to learn!

**Comment by RVany** [comment]
> Nice! Got those other things sorted out. Unfortunately, SkillAccordion stories appear to have broken as a result of changes to fakeExperiences
> 
> image

Its working for me!
image

**Review by petertgiles — CHANGES_REQUESTED** [review]
I see the broken skill accordions as well.  Since you've only touched the experience accordions it was probably not your fault. :smile:  Could you try to fix it while you're working on these, though?  If it turns out to be a big problem please open an issue on the board for it.

Also, could you use [the date formatting function that Brinda worked out](https://github.com/GCTC-NTGC/gc-digital-talent/blob/ca974164d5e50cb78d768aa8b38f964253700c3e/frontend/talentsearch/src/js/components/skills/SkillAccordion/SkillAccordion.tsx#L35)?  It's not as accurate to the wireframe but I think it's superior since it will localize months in French.  And they two types of accordions should probably match. :laughing:   You could copy/paste it but the better solution would probably be to move it to one of the common helper function files and update both accordion files to point to the shared location.

**Comment by petertgiles** [comment]
> @petertgiles Can you please share the wireframe?

https://www.figma.com/proto/PrvXEDac0HVp7dF7SLWVFm/Candidate-profile-interfaces---V5.0.0?node-id=2340%3A4769&starting-point-node-id=1%3A2

**Comment by RVany** [comment]
> 

Ok I try to do that. Thank you!

**Comment by vd1992** [comment]
> I see the broken skill accordions as well. Since you've only touched the experience accordions it was probably not your fault. 😄 Could you try to fix it while you're working on these, though? If it turns out to be a big problem please open an issue on the board for it.
> 
> Also, could you use [the date formatting function that Brinda worked out](https://github.com/GCTC-NTGC/gc-digital-talent/blob/ca974164d5e50cb78d768aa8b38f964253700c3e/frontend/talentsearch/src/js/components/skills/SkillAccordion/SkillAccordion.tsx#L35)? It's not as accurate to the wireframe but I think it's superior since it will localize months in French. And they two types of accordions should probably match. 😆 You could copy/paste it but the better solution would probably be to move it to one of the common helper function files and update both accordion files to point to the shared location.

FakeExperiences was updated, Skill Accordion draws from that, so my guess was a tweak to that faker inadvertently broke Skill Accordion as that component needed things in a very specific way. 
All I see is more reason we need that Chromatic comparison thing to avoid something being broken in who-knows-what place and no one notices for weeks

**Comment by RVany** [comment]
> 
> > I see the broken skill accordions as well. Since you've only touched the experience accordions it was probably not your fault. 😄 Could you try to fix it while you're working on these, though? If it turns out to be a big problem please open an issue on the board for it.
> > Also, could you use [the date formatting function that Brinda worked out](https://github.com/GCTC-NTGC/gc-digital-talent/blob/ca974164d5e50cb78d768aa8b38f964253700c3e/frontend/talentsearch/src/js/components/skills/SkillAccordion/SkillAccordion.tsx#L35)? It's not as accurate to the wireframe but I think it's superior since it will localize months in French. And they two types of accordions should probably match. 😆 You could copy/paste it but the better solution would probably be to move it to one of the common helper function files and update both accordion files to point to the shared location.
> 
> FakeExperiences was updated, Skill Accordion draws from that, so my guess was a tweak to that faker inadvertently broke Skill Accordion as that component needed things in a very specific way. All I see is more reason we need that Chromatic comparison thing to avoid something being broken in who-knows-what place and no one notices for weeks

How I couln't see any broken accordions. Anyway, I undo the changes in fakeExperiences

**Comment by vd1992** [comment]
> > > I see the broken skill accordions as well. Since you've only touched the experience accordions it was probably not your fault. 😄 Could you try to fix it while you're working on these, though? If it turns out to be a big problem please open an issue on the board for it.
> > > Also, could you use [the date formatting function that Brinda worked out](https://github.com/GCTC-NTGC/gc-digital-talent/blob/ca974164d5e50cb78d768aa8b38f964253700c3e/frontend/talentsearch/src/js/components/skills/SkillAccordion/SkillAccordion.tsx#L35)? It's not as accurate to the wireframe but I think it's superior since it will localize months in French. And they two types of accordions should probably match. 😆 You could copy/paste it but the better solution would probably be to move it to one of the common helper function files and update both accordion files to point to the shared location.
> > 
> > 
> > FakeExperiences was updated, Skill Accordion draws from that, so my guess was a tweak to that faker inadvertently broke Skill Accordion as that component needed things in a very specific way. All I see is more reason we need that Chromatic comparison thing to avoid something being broken in who-knows-what place and no one notices for weeks
> 
> How I couldn't see any broken accordions. Anyway, I undo the changes in fakeExperiences

_Skill_ Accordions, you posted a picture of _Experience_ Accordions above. Different component

**Comment by RVany** [comment]
> 



> > > > I see the broken skill accordions as well. Since you've only touched the experience accordions it was probably not your fault. 😄 Could you try to fix it while you're working on these, though? If it turns out to be a big problem please open an issue on the board for it.
> > > > Also, could you use [the date formatting function that Brinda worked out](https://github.com/GCTC-NTGC/gc-digital-talent/blob/ca974164d5e50cb78d768aa8b38f964253700c3e/frontend/talentsearch/src/js/components/skills/SkillAccordion/SkillAccordion.tsx#L35)? It's not as accurate to the wireframe but I think it's superior since it will localize months in French. And they two types of accordions should probably match. 😆 You could copy/paste it but the better solution would probably be to move it to one of the common helper function files and update both accordion files to point to the shared location.
> > > 
> > > 
> > > FakeExperiences was updated, Skill Accordion draws from that, so my guess was a tweak to that faker inadvertently broke Skill Accordion as that component needed things in a very specific way. All I see is more reason we need that Chromatic comparison thing to avoid something being broken in who-knows-what place and no one notices for weeks
> > 
> > 
> > How I couldn't see any broken accordions. Anyway, I undo the changes in fakeExperiences
> 
> _Skill_ Accordions, you posted a picture of _Experience_ Accordions above. Different component

Because Experience Accordions is the issue in this PR. But Skill Accordions also working for me both locally and remote. I even shared the screen to Peter. Check again please. I am trying to find out how I can format the date in 
Experince Acordinons without touching the fakeExperiences. Also that is not a proper way!

**Comment by RVany** [comment]
> * experience accordion looks good
> * skill accordion back to normal
> 
> I think all the tasks are taken care of, good work. However, please take a look at the Jest test that is failing. Looks like the skillAccordion test, please update that so it works with your new date style

ok I will take a look. I never touch the test though!

**Comment by vd1992** [comment]
the test probably broke because of formatting changes elsewhere, I think you just need to change the tests around a bit so they don't fail

**Comment by RVany** [comment]
> the test probably broke because of formatting changes elsewhere, I think you just need to change the tests around a bit so they don't fail

ok. Thank you!

File: frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx [hunk 2]
```diff
@@ -38,22 +36,12 @@ const CommunityAccordion: React.FunctionComponent<CommunityExperience> = ({
     <Accordion
       title={intl.formatMessage(
         {
-          defaultMessage: "{title} at {organization}",
+          defaultMessage: "{title} of {organization}",
```
Comment (petertgiles) [inline frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx:39]: Was this change intentional?  I'm having trouble finding Jerbo's post that had all the individual accordions.

Comment (petertgiles) [reply frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx:39]: Found it!  Community should be _at_ not _of_.  https://github.com/GCTC-NTGC/gc-digital-talent/issues/1925#issuecomment-1054640559

Comment (RVany) [reply frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx:39]: I don't think I change it.

Comment (petertgiles) [reply frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx:39]: You changed it in commit https://github.com/GCTC-NTGC/gc-digital-talent/pull/2454/commits/abb62a0633834bc3788d761179cb52d2adb549cf .  Hehe, git knows everything!

Comment (RVany) [reply frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx:39]: really? why I did that?

Comment (RVany) [reply frontend/talentsearch/src/js/components/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx]: I changed it back.

**Comment by RVany** [comment]
Now, the jest test is passed but date format in `frontend\common\src\fakeData\fakeExperiences.ts` is not matching the component's date format though.

**Comment by petertgiles** [comment]
> s not matching the component's date format though.

The javascript Date parser is good enough to understand the date strings from both faker and the API so it's working OK. :smile:

**Comment by RVany** [comment]
> > s not matching the component's date format though.
> 
> The javascript Date parser is good enough to understand the date strings from both faker and the API so it's working OK. 😄

Ok! Thank you!😀
